rocAL/rocAL_pybind/examples/rocAL_example_doc/resize_implementation.ipynb

```diff
@@ -0,0 +1,202 @@
+            "id": "e0569dcc",
+            "metadata": {},
+            "source": [
+                "<font size= \"12\" >Configuring rocAL pipeline </font>"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "00c2815c",
+            "metadata": {},
+            "source": [
+                "Configure the pipeline parameters as required by the user."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "id": "f118bb1f",
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "data_path = \"/media/MIVisionX-data/rocal_data/coco/coco_10_img/train_10images_2017/\"\n",
+                "rocal_cpu = True\n",
+                "device = \"cpu\"\n",
+                "batch_size = 1\n",
+                "num_threads = 1\n",
+                "random_seed = 1\n",
+                "local_rank = 0\n",
+                "world_size = 1\n",
+                "display = True"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "id": "7e069c4d",
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "pipe = Pipeline(batch_size=batch_size, num_threads=num_threads, device_id=local_rank, seed=random_seed, rocal_cpu=rocal_cpu, tensor_layout=types.NHWC, tensor_dtype=types.FLOAT)\n"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "df5e6005",
+            "metadata": {},
+            "source": [
+                "<font size=\"12\"> Image augmentation pipeline </font>"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "e1ab279e",
+            "metadata": {},
+            "source": [
+                "Here we use the file reader followed by the turbo jpeg decoder. In this pipeline we add cascaded augmentation on the decoded image.<br>We enable the output for differnet augmentaion using set_output"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "id": "804d8895",
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "resize_w = 200\n",
+                "resize_h = 200\n",
+                "with pipe:\n",
+                "        jpegs, _ = fn.readers.file(file_root=data_path)\n",
+                "        images = fn.decoders.image(jpegs, file_root=data_path, device=0, output_type=types.RGB, shard_id=0, num_shards=1, random_shuffle=False)\n",
+                "        output_linear = fn.resize(images, resize_width=resize_w, resize_height=resize_h, interpolation_type=types.LINEAR_INTERPOLATION)\n",
+                "        output_nearest_neighbor = fn.resize(images, resize_width=resize_w, resize_height=resize_h, interpolation_type=types.NEAREST_NEIGHBOR_INTERPOLATION)\n",
```

Comment:
I think this script is wrong. Looks like fn.resize is called repeatedly for all augmentations

Reply:
This is an example notebook explaining the resize augmentation with various interpolation types. The pipeline has multiple outputs from resize augmentation with various interpolation types